documentation/docs/libraries/nodejs/examples.mdx

```diff
@@ -1,3 +1,19 @@
+import CodeBlock from '@theme/CodeBlock';
+import get_info from '!!raw-loader!./../../../../bindings/nodejs/examples/01_get_info.js';
+import get_info_b from '!!raw-loader!./../../../../bindings/nodejs/examples/01b_get_info.js';
```

Comment:
A little bit inconsistent to have `get_info_b` and then `a_get_address_outputs`, `b_get_output`